Framework/PythonInterface/plugins/algorithms/DPDFreduction.py

```diff
@@ -370,6 +390,32 @@ def PyExec(self):
 
         self.setProperty("OutputWorkspace", api.mtd[wn_sqes])
 
+    def _load(self, run_numbers, data_name):
+        """
+        Load data and monitors for run numbers and monitors.
+        Algorithm 'Load' can aggregate many runs into a single workspace, but it is not able to do so
+        with the monitor workspaces.
+        :param run_numbers: run numbers for data event files
+        :param data_name: output name for data workspace. The name for the workspace holding the
+        monitor data will be data_name+'_monitors'
+        :return: None
+        """
+        # Find out the files for each run
+        load_algorithm = AlgorithmManager.createUnmanaged("Load")
+        load_algorithm.initialize()
+        load_algorithm.setPropertyValue('Filename', str(run_numbers))
+        files = (load_algorithm.getProperty('Filename').value)[0]
+        if not isinstance(files, list):
+            # run_numbers represents one file only
+            api.Load(Filename=files, LoadMonitors=True, OutputWorkspace=data_name)
```

Comment:
You can use `LoadEventNexus` from here on, if you only want this to work on ARCS. Leave `Load` if you want this to work at other facilities